traefik/templates/requirements.yaml

```
@@ -1,6 +1,6 @@
 {{- $version := include "imageVersion" $ }}
 {{- if (ne $version "experimental-v3.0") }}
-  {{- if (semverCompare "<v3.0.0-0" $version) }}
+  {{- if (semverCompare "<v3.1.0-0" $version) }}
```

````suggestion
  {{- if (semverCompare "<v3.0.0-0" $version) }}
````
We will keep v3+ compatiblity FTM.
The requirements for v3.1+ is only for safeQueryParams
Something like this, with the corresponding test:

```go
{{- if and (semverCompare "<v3.1.0-0" $version) .Values.tracing.safeQueryParams }}
  {{ fail ...}}
{{- end }}
```